operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -0,0 +1,475 @@
+# Add NCN Data
+
+## Description
+
+Add NCN data to System Layout Service (SLS), Boot Script Service (BSS), and Hardware State Manager (HSM) as needed, in order to add an NCN to the system.
+
+Scenarios where this procedure is applicable:
+
+1. Adding a management NCN that has not previously been in the system before:
+   * Add an additional NCN to an existing cabinet
+   * Add an NCN that is being replaced by another NCN of the same type and in the same slot
```

Comment:
Should this be "Adding an NCN that is replacing another NCN of the same type and in the same slot"?
Otherwise, why are we adding an NCN that is also being replaced?